### DISCUSSION TITLE
Onboarding rules

### DISCUSSION THREAD

**Discussion by dor272** [renovatebot/renovate#19121] [Request Help]
Hi,

I'm setting up Renovate on a GitLab group, and I'm wondering if there's a way to set different onboarding rules for different sub-groups.
For example, I would want that by default PRs/MRs in subgroup-x to be assigned to X and those in subgroup-y to Y.

Thank you.

**Comment by rarkins** [comment]
The preferred way is that the repos define a `renovate.json` file in their repo to control their own config.

If you want to configure things in the bot's own config, then you can use the same approach as I gave here: https://github.com/renovatebot/renovate/discussions/19031#discussioncomment-4204696

**Reply by dor272** [reply]
I understand that I can change a repo behavior with its `renovate.json`.
As far as I understand, your suggested approach still requires me to hard code the repos names I want to adjust. Or can I use a minimatch/regex at the `repository` key?

My main idea is to have different presets for different subgroups (managed in a repo). I would like to use `autodiscover` to set different onboarding configs based on a repo subgroup while having a single repo to manage all the presets and to run the CI.

To my understanding, it is not possible, I will either have to:
1. Split the CI so each CI will have its own `autodiscoverFilter`
2. Manually maintain a repo list in the onboarding conifg
3. Manually onboard new repos

**Reply by rarkins** [reply]
Currently the list of repository names needs to be an exact match - no pattern/regex. If you're interested in contributing a feature to improve this, let's chat
